src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Geometry/BufferList/BufferList.xaml.cs

```diff
@@ -7,149 +7,240 @@
+        private void Initialize()
+        {
+            // Create a spatial reference that's suitable for creating planar buffers in north central Texas (State Plane).
+            SpatialReference statePlaneNorthCentralTexas = new SpatialReference(32038);
 
-            // Set the map's initial extent to the envelope.
-            theMap.InitialViewpoint = new Viewpoint(startingEnvelope);
+            // Define a polygon that represents the valid area of use for the spatial reference.
+            // This information is available at https://developers.arcgis.com/net/latest/wpf/guide/pdf/projected_coordinate_systems_rt100_3_0.pdf
+            List<MapPoint> spatialReferenceExtentCoords = new List<MapPoint>
+            {
+                new MapPoint(-103.070, 31.720, SpatialReferences.Wgs84),
+                new MapPoint(-103.070, 34.580, SpatialReferences.Wgs84),
+                new MapPoint(-94.000, 34.580, SpatialReferences.Wgs84),
+                new MapPoint(-94.00, 31.720, SpatialReferences.Wgs84)
+            };
+            _spatialReferenceArea = new Polygon(spatialReferenceExtentCoords);
+            _spatialReferenceArea = GeometryEngine.Project(_spatialReferenceArea, statePlaneNorthCentralTexas) as Polygon;
+
+            // Create a map that uses the North Central Texas state plane spatial reference.
+            Map bufferMap = new Map(statePlaneNorthCentralTexas);
+
+            // Add some base layers (counties, cities, and highways).
+            Uri usaLayerSource = new Uri("https://sampleserver6.arcgisonline.com/arcgis/rest/services/USA/MapServer");
+            ArcGISMapImageLayer usaLayer = new ArcGISMapImageLayer(usaLayerSource);
+            bufferMap.Basemap.BaseLayers.Add(usaLayer);
+
+            // Use a new EnvelopeBuilder to expand the spatial reference extent 120%.
+            EnvelopeBuilder envBuilder = new EnvelopeBuilder(_spatialReferenceArea.Extent);
+            envBuilder.Expand(1.2);
+
+            // Set the map's initial extent to the expanded envelope.
+            Envelope startingEnvelope = envBuilder.ToGeometry();
+            bufferMap.InitialViewpoint = new Viewpoint(startingEnvelope);
 
             // Assign the map to the MapView.
-            MyMapView.Map = theMap;
+            MyMapView.Map = bufferMap;
+
+            // Create a graphics overlay to show the buffer polygon graphics.
+            GraphicsOverlay bufferGraphicsOverlay = new GraphicsOverlay
+            {
+                // Give the overlay an ID so it can be found later.
+                Id = "buffers"
+            };
 
-            // Create a graphics overlay to show the buffer-related graphics.
-            _graphicsOverlay = new GraphicsOverlay();
+            // Create a graphic to show the spatial reference's valid extent (envelope) with a dashed red line.
+            SimpleLineSymbol lineSymbol = new SimpleLineSymbol(SimpleLineSymbolStyle.Dash, System.Drawing.Color.Red, 5);
```

Comment:
This can be simplified to `Color.Red` because `System.Drawing` is imported above.